src/main/java/com/jvm_bloggers/core/newsletter_issues/domain/NewsletterIssue.java

```diff
@@ -58,6 +61,7 @@
     @Singular
     @NonNull
     @OneToMany(fetch = FetchType.EAGER)
+    @Fetch(value = FetchMode.SUBSELECT)
```

Comment:
Probably if you change List to Set (do we really need List here) problem is gone without adding this annotation.